frontend/packages/dev-console/src/components/topology/shapes/WorkloadNode.tsx

```diff
@@ -33,7 +30,7 @@ const WorkloadNode: React.FC<NodeProps<WorkloadData>> = ({
       x={x}
       y={y}
       outerRadius={radius}
-      innerRadius={radius * 0.55}
+      innerRadius={radius * 0.45}
```

Comment:
Can we gate the shrinkage of the center image around the existence of the second donut? I don't think we need to shrink the icon always - I'm not sure how often rollout will be in-use... but the standing look is definitely the first impression people will see of the node and a smaller icon isn't helping anything.

Reply:
fixed in https://github.com/openshift/console/pull/2496/commits/6353ee79b77029607673e09df07a9099b0fb4a9b